package.json

```diff
@@ -1,42 +1,24 @@
 {
-  "name": "ember-bem-helpers",
-  "version": "0.4.0",
-  "description": "BEM helpers for Ember.js applications",
-  "keywords": [
-    "ember-addon",
-    "BEM",
-    "helper"
-  ],
   "repository": {
     "type": "git",
     "url": "git+https://github.com/retailnext/ember-bem-helpers.git"
   },
   "license": "MIT",
   "author": "RetailNext",
-  "directories": {
-    "lib": "lib",
-    "test": "tests"
-  },
   "scripts": {
-    "build": "ember build --environment=production",
-    "lint": "concurrently \"pnpm:lint:*(!fix)\" --names \"lint:\"",
-    "lint:css": "stylelint \"**/*.css\"",
-    "lint:css:fix": "concurrently \"pnpm:lint:css -- --fix\"",
-    "lint:fix": "concurrently \"pnpm:lint:*:fix\" --names \"fix:\"",
-    "lint:hbs": "ember-template-lint .",
-    "lint:hbs:fix": "ember-template-lint . --fix",
-    "lint:js": "eslint . --cache",
-    "lint:js:fix": "eslint . --fix",
-    "start": "ember serve",
-    "test": "concurrently \"pnpm:lint\" \"pnpm:test:*\" --names \"lint,test:\"",
-    "test:ember": "ember test",
-    "test:ember-compatibility": "ember try:each",
-    "test:ember-try-one": "ember try:one"
+    "build": "pnpm --filter ember-bem-helpers build",
+    "lint": "pnpm --filter '*' lint",
+    "prepare": "pnpm build",
+    "start": "concurrently 'npm:start:*' --restart-after 5000 --prefix-colors cyan,white,yellow",
+    "start:addon": "pnpm --filter ember-bem-helpers start --no-watch.clearScreen",
+    "start:test-app": "pnpm --filter test-app start",
+    "test": "pnpm --filter '*' test",
+    "test:ember": "pnpm --filter '*' test:ember"
   },
   "dependencies": {
-    "@babel/core": "^7.23.6",
-    "ember-cli-babel": "^8.2.0",
-    "ember-cli-htmlbars": "^6.3.0"
+    "@babel/core": "7.23.7",
+    "ember-cli-babel": "8.2.0",
+    "ember-cli-htmlbars": "6.3.0"
```

Comment:
Are you supposed to have dependencies and dev dependencies in the root package.json? How are they used?

Reply:
Removed